src/task/methods/scape/script.py

```diff
@@ -124,8 +146,9 @@
 for i, d in enumerate(top_drugs):
 		print(i, d)
 		scm = scape.model.create_default_model(par["n_genes_enhanced"], df_de_c, df_lfc_c)
+		cell = confirm_celltype(par["cell"], d)
```

Comment:
if the other suggestion is accepted

```suggestion
		cell = confirm_celltype(df_de, par["cell"], d)